reducer/src/main/java/com/graphicsfuzz/reducer/reductionopportunities/LiteralToUniformReductionOpportunity.java

```diff
@@ -25,18 +30,40 @@
 
   private final Expr literalExpr;
   private final PipelineInfo pipelineInfo;
+  private final TranslationUnit translationUnit;
+  private final int index;
```

Comment:
This field deserves a short comment to explain what it indexes into.

Reply:
Done.